src/Sylius/Behat/Context/Setup/TaxonomyContext.php

```diff
@@ -148,6 +170,29 @@ private function createTaxon($name)
     }
 
     /**
+     * @param array $names
+     *
+     * @return TaxonInterface
+     */
+    private function createTaxonInManyLanguages(array $names)
+    {
+        /** @var TaxonInterface $taxon */
+        $taxon = $this->taxonFactory->createNew();
+        $taxon->setCode(StringInflector::nameToCode($names['en_US']));
```

Comment:
It will :boom: in a massive explosion that would be visible from the Moon! We could obviously pick there name in first locale.